### PR TITLE
Renaming Variables

### PR DESCRIPTION
**Pull request by Shetty1103**
This PR
1. Provides more self-explanatory/explicit names(to be recognized by someone working on GNSS) for the parameters computed by PRX.
2. Added a test to verify the renaming.

### CODE REVIEW

**Comment by jtec** [comment]
@Shetty1103 Thanks for taking this on! When the PR is ready for review, simply take it out of draft mode and ping @plutonheaven or me as reviewers :)

**Comment by plutonheaven** [comment]
@jtec, I had to delete `diskcache`, and after that, all `pytest` worked. I think that the earlier we merge this into the main branch, the better, to avoid future conflicts if we keep the current variable names.

File: src/prx/main.py [hunk 1]
```diff
@@ -78,16 +78,16 @@ def write_csv_file(
     # write header
     with open(output_file, "w", encoding="utf-8") as file:
         file.write(f"# {json.dumps(prx_header)}\n")
-    flat_records["elevation_deg"] = np.rad2deg(flat_records.elevation_rad.to_numpy())
-    flat_records["azimuth_deg"] = np.rad2deg(flat_records.azimuth_rad.to_numpy())
+    flat_records["sat_elevation_deg"] = np.rad2deg(flat_records.elevation_rad.to_numpy())
+    flat_records["sat_azim_deg"] = np.rad2deg(flat_records.azimuth_rad.to_numpy())
```
Comment (jtec) [inline src/prx/main.py:82]: nit: suggest `sat_azim_deg` here to be consistent with `sat_elevation_deg` - or change to `sat_elev_deg` :)

File: src/prx/main.py [hunk 2]
```diff
@@ -361,12 +361,12 @@ def format_flat_rows(row):
...
     sat_specific = sat_states[
         sat_states.columns.drop(
-            ["observation_type", "group_delay_m", "time_of_reception_in_receiver_time"]
+            ["observation_type", "sat_instrumental_delay_m", "time_of_reception_in_receiver_time"]
         )
     ].drop_duplicates(subset=["satellite", "time_of_emission_isagpst"])
     # Group delays are signal-specific, so we merge them in separately
     code_specific = sat_states[
-        ["satellite", "observation_type", "time_of_emission_isagpst", "group_delay_m"]
+        ["satellite", "observation_type", "time_of_emission_isagpst", "sat_instrumental_delay_m"]
```
Comment (jtec) [inline src/prx/main.py:369]: would `broadcast_code_bias` be an appropriate name here?

File: src/prx/test/test_main.py [hunk 3]
```diff
@@ -115,3 +118,48 @@ def test_spp_lsq(input_for_test):
             < 1e1
         )
         assert np.max(np.abs(vt_lsq[0:3, :])) < 1e-1
+
+
+def test_csv_parameter_renaming(input_for_test):
```
Comment (jtec) [inline src/prx/test/test_main.py:123]: Suggest 'test_csv_column_names'

File: src/prx/test/test_main.py [hunk 4]
```diff
@@ -115,3 +118,48 @@ def test_spp_lsq(input_for_test):
...
+    expected_prx_file = Path(
+        str(test_file).replace("crx.gz", constants.cPrxCsvFileExtension)
+    )
+    assert expected_prx_file.exists()
+
+    # Read the CSV file
+    df = pd.read_csv(expected_prx_file, comment="#")
+
+    # Renamed parameters
+    renamed_parameters = {
```
Comment (jtec) [inline src/prx/test/test_main.py:135]: suggest 

```
# Expected CSV column names
expected_column_names = {
...
```

**Review by jtec — APPROVED** [review]
Looks good too me! Left just a few minor comments.

**Comment by jtec** [comment]
@Shetty1103 Could you add a short description to the PR? Just a few words that summarise what it does and why.

File: src/prx/main.py (hunk 1 above)
Comment (plutonheaven) [reply src/prx/main.py:82]: I'd favor `sat_azimuth_deg`

File: src/prx/main.py (hunk 2 above)
Comment (plutonheaven) [reply src/prx/main.py:369]: There are many possible models related to this parameter: differential code bias or total/timing/broadcast group delay.
Those terms are referred to as satellite instrumental delays in the ESA book, hence the proposal.
I think in the long term, we will have to account for satellite code and phase bias. We could then go for sat_code_bias_m for the moment?

**Comment by plutonheaven** [comment]
Hi @Shetty1103, we made a few modifications to the `main` branch. You have to merge again the `main` branch into your own branch, solve conflict and push the update.
Also, let's use `sat_code_bias_m` for the instrumental delay.
Once it's approved, you can use the `squash and merge` button on this page, to keep only a short relevant commit message.

**Pull request merged**